server/routes/auth.js

```diff
@@ -42,7 +42,9 @@ router.post("/register", validationMiddleware, async function (req, res, next) {
         // verify that user isn't already in DB
```

Comment:
Currently you can choose if you want to be a chef on the sign up form.  If we want to keep that feature I believe we'll need to add a check for that flag and save that user to the chef collection and update isChef on the user (it currently defaults to false).

Reply:
Yes, we can do that but we don't have to flow on the front-end to capture all the chef details needed for the creation of chef profile.

Reply:
I suppose we don't need the cuisine specialty to be required right?

Reply:
You're right. Let me update the API to make it happen.

Reply:
I have updated it so that now isChef is set for user when registering